node/pod.go

```diff
@@ -267,15 +267,28 @@ func (pc *PodController) updatePodStatus(ctx context.Context, pod *corev1.Pod) e
 			pod.Status.Reason = "NotFound"
 			pod.Status.Message = "The pod status was not found and may have been deleted from the provider"
 			for i, c := range pod.Status.ContainerStatuses {
+				var startedAt metav1.Time
+				var fininshedAt metav1.Time
+				if c.State.Waiting != nil {
+					// Skip waiting state
+					continue
```

Comment:
Doesn't this result in a behaviour change for waiting containers? Also, what's going on that results in this crash?

Reply:
#583 describes the crash in detail. `c.State.Running` is `nil` when one of the containers is in waiting state, causing a crash when VK tries to access `c.State.Running.StartedAt`. What type of behavior change are you expecting from this change?